(CONT-589) - Updating README.md with Deferred Function for sensitive fields (#437)
puppetlabs/puppetlabs-java_ks: README.md

java_ks { 'broker.example.com:/etc/activemq/broker.ks':
  ensure              => latest,
  certificate_content => $certificate_data_fetched_from_secure_store,
  private_key_content => $private_key_data_fetched_from_secure_store,
  password            => 'albatros',
  password_fail_reset => true,
}
```

> **NOTE:** The sensitive fields like `password`, `certificate_content` and `private_key_content` can be deferred using the [Deferred](https://www.puppet.com/docs/puppet/7/template_with_deferred_values.html) function on Puppet Master and enable to execute on agent. This will avoid the sensitive values to be part of Puppet Catalog.

Reviewer: ```suggestion
**NOTE:** Sensitive fields like `password`, `certificate_content` and `private_key_content` can be deferred using the [Deferred](https://www.puppet.com/docs/puppet/7/template_with_deferred_values.html) function. This will ensure sensitive values are not present in the Catalog.
```
